src/tests/frontend/frontend-utilities.test.ts

```diff
@@ -165,20 +165,20 @@ describe("longFormatDate", () => {
     ["fr-CA", "23 juillet 2023 à 13 h 31"],
     ["fr-FR", "23 juillet 2023 à 13:31"],
     ["he-IL", "23 ביולי 2023 בשעה 13:31"],
-    ["hi-IN", "23 जुलाई 2023 को 1:31 pm"],
-    ["id-ID", "23 Juli 2023 13.31"],
-    ["it-IT", "23 luglio 2023 13:31"],
+    ["hi-IN", "23 जुलाई 2023 को 1:31 pm बजे"],
+    ["id-ID", "23 Juli 2023 pukul 13.31"],
+    ["it-IT", "23 luglio 2023 alle ore 13:31"],
```

Comment:
Confused about these.  These tests pass on my box and in the Github actions.  Are they not passing for you?